src/typeDefs/root.js

```diff
@@ -1,10 +1,11 @@
 import { gql } from "apollo-server-express";
-import { GraphQLScalarType } from 'graphql';
-import { Kind } from 'graphql/language';
+import { GraphQLScalarType } from "graphql";
+import { Kind } from "graphql/language";
 
 export default gql`
   scalar Date
-  
+  scalar Upload
```

Comment:
i guess this is for file upload
but we are using File?

Reply:
This is required when we use schema + middleware

Reply:
ok